docs/ebpf/guides/getting-started.md

```diff
@@ -0,0 +1,311 @@
+# Creating an eBPF-powered Go application
+
+!!! tip ""
```

Comment:
As discussed on our call, I think this whole `tip` should be on the home page. Both to set the expectations of the reader and to give us a guideline what kind of person to write for.